### ISSUE TITLE
fullscreenchange native

### ISSUE THREAD

**Issue by mjau-mjau** [sindresorhus/screenfull.js#38]
I was expecting fullscreenchange event to fire on native fullscreen method from the browser menu, but it doesn't, at least not in Chrome. This makes it difficult to manage interface elements if the visitors goes into fullscreen from the browser menu ... I assume this is something unsolvable? The following triggers when toggling full screen from JS, but not when entering fullscreen from browser menu.

``` javascript
$(document).on(screenfull.raw.fullscreenchange, function(){
  console.log(screenfull.isFullscreen);
});
```

Basically I can't toggle my expand/compress button icons properly. It is strange, because the fullscreen method from browser menu does _exit_ fullscreen, and then the event fires, but not when entering.
